tokio-util/src/io/copy_to_bytes.rs

```diff
@@ -66,3 +67,14 @@ where
         self.project().inner.poll_close(cx)
     }
 }
+
+impl<S, T> Stream for CopyToBytes<S>
+where
+    S: Stream<Item = T> + std::marker::Unpin,
+{
+    type Item = T;
+    fn poll_next(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
+        let ref_mut = self.get_mut().get_mut();
+        Pin::new(ref_mut).poll_next(cx)
+    }
+}
```

Comment:
We don't want `Unpin` bounds if they're not necessary. Also, the `T` parameter can be dropped.
```suggestion
impl<S: Stream> Stream for CopyToBytes<S> {
    type Item = S::Item;
    fn poll_next(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
        self.project().inner.poll_next(cx)
    }
}
```

Reply:
fixed

Reply:
Please also fix it for `SinkWriter` and `StreamReader`.

Reply:
done